### ISSUE TITLE
documentation fails to build

### ISSUE THREAD

**Issue by cz4rs** [DARMA-tasking/checkpoint#173]
Documentation fails to build on `develop`:
```
CMake Error at CMakeLists.txt:56 (include):
  include could not find load file:

    cmake/load_doxygen.cmake
```
The file is available in `vt` at the expected location.

**Reproduction**:
`docker-compose run ubuntu-docs`

**Note**:
Make sure that `vt` builds correctly after fixing this.

**Comment by cz4rs** [comment]
the last run happened on Dec 3 and failed as described above:
![Screenshot from 2021-01-19 00-02-43](https://user-images.githubusercontent.com/4639165/104969787-be4e8100-59e9-11eb-9d02-b64e54a817f2.png)

- ~`docker-compose run ubuntu docs` works just fine in local~
- ~looking at the repository, `load_doxygen.cmake` is in correct place~
- ~`Build Documentation` job is triggered by a push to develop, so I cannot verify this manually, but it should work as expected after next PR is merged~

edit: it still doesn't work, I have messed up my local clone and got the build to pass somehow

**Comment by cz4rs** [comment]
One thing to note: `DARMA-tasking.github.io` repository is almost 700 MB at the moment:
```bash
DARMA-tasking.github.io > du -sh .
697M	.
```
with the biggest offenders being `docs/xml/index.xml` and `docs/html/searchdata-v1.js`.
Their history could be removed with a tool like [BFG Repo-Cleaner](https://rtyley.github.io/bfg-repo-cleaner/) if needed.
